### DISCUSSION TITLE
更新日志 - 22.03.5 - 20231208

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#1164] [Announcements]
# 系统
默认不开启WAN口访问，以避免运营商DHCP联网的情况下被攻击（国内运营商一般都是PPPoE）。 #1132
wan_drop机制（PPPoE自动禁止WAN访问）依然保留，以免小白用户错误将设备开放到公网，如确实需要开放部分或全部端口，可以在防火墙的通信规则中添加相应规则（支持端口范围）。 
以上变更只对全新刷机或恢复出厂设置后有效，对于单网口设备，默认就是LAN口，不受此变更影响。
简单来说，如果全新刷机或恢复出厂设置后，需要通过LAN口才能访问后台。

温馨提示：不管怎样，都建议修改默认密码，尤其是跟公网连接的x86设备（因为除安卓病毒外大部分病毒都是针对x86平台的），已经出现过机器被挂木马或挖矿程序的案例。

# 设备
* RK35xx:
  1. 支持了HDMI输出，大部分有HDMI接口的设备应该都有输出了（默认没有应用，只会显示黑屏和一个文本终端光标）
  2. RK3588 支持了GPU #1121

**Comment by jjm2473** [comment]
R5S/R5C的HDMI失效，下个版本会修复

**Comment by qjmyymq** [comment]
下载固件速度特别慢

**Reply by jannson** [reply]
手动下载呢？可以手动下载一个，手动下载应该会走边缘节点的。要不就是我们的边缘节点掉线了

**Comment by jangjoan** [comment]
这个版本更新固件下载速度非常慢

**Comment by jangjoan** [comment]
已经下载完更新上了，谢谢。就是比平常更新下载时间要长发自我的 iPhone在 2023年12月9日，11:28，Janson ***@***.***> 写道：﻿
手动下载呢？可以手动下载一个，手动下载应该会走边缘节点的。要不就是我们的边缘节点掉线了

**Comment by ysjy11** [comment]
看到大家都说下载慢，我就放心了，还以为自己网络出现了问题

**Comment by Clxj** [comment]
下载慢

**Comment by Aria-0** [comment]
下载特别慢，要下载将近一个小时。

**Comment by bizshow** [comment]
不管是在线安装，还是下载了手动离线安装这个固件，安装结束后自动重启，还是旧版本，感觉没有安装成功

**Reply by jjm2473** [reply]
如果有串口或者x86有显示器，最好看看输出，是不是哪些程序无法关闭

**Comment by carolchenme** [comment]
固件更新模版地址：
https://www.chuangkit.com/designtools/self/design
<img width="1349" alt="image" src="https://github.com/istoreos/istoreos/assets/26473337/a3466746-eaed-4b88-8f0e-adf4c9c53914">

**Comment by William0606** [comment]
还好，不到五分钟就下好了
